src/main/webapp/WEB-INF/tags/admin/navBar.tag

```diff
@@ -22,9 +22,10 @@
                     <a href="<%=Const.ActionURIs.ADMIN_HOME_PAGE%>">Create Instructor</a>
                 </li>
                 
+                <!-- The link to Account Management Page will be hidden until it's scalable.
```

Comment:
I think it won't be that far. Just for 2-3 weeks until I finish all pending PRs. So I want to keep the link there in order to recover it easier.